index.html

```diff
@@ -4,9 +4,126 @@
+    <div class="address">
+        <h4>@Main Road Vanike Tararr, Hafizabad</h4>
+    </div>
+</section>
+<section id="dishes">
+    <div class="dishes-heading"><h2>Main Dishes</h2></div>
+    <div class="orangeline"></div>
+    <div class="dish-details">
+    <div class="dish"><img src="./images/grill-fish.png" alt="Grill Fish">
+    <h3>Grill Fish</h3>
+    <p>Enjoy Fish on the Grill With These Top Recipes · Ginger-Lime Swordfish · Salmon With Lemon and Thyme · Sea Bass With Garlic Butter.</p>
+    </div>
+    <div class="dish"><img src="./images/crown-crust-pizza.png" alt="Crown crust pizza">
+        <h3>Crust Pizza</h3>
+        <p>The Crown Crust Pizza is a blend of desi Chicken Filling, and topped special Extreme Pizza Sauce , veggies and less cheese.</p>
+        </div>
+        <div class="dish"><img src="./images/steak-sandwich.png" alt="Steak Sandwich">
+            <h3>Steak Sandwich</h3>
+            <p>Sandwich prepared with steak that has been broiled, fried, grilled, barbecued or seared using steel grates or gridirons, then served on bread or a roll.</p>
+            </div>  
+            <div class="dish"><img src="./images/chicken-steak-burger.png" alt="chicken-steak-burger">
+                <h3>Steak Burger</h3>
+                <p>Chicken Steak is your saviour in all such times. These burger patties consist of whole chicken breasts that are seasoned with various spices.</p>
+                </div> 
+                <div class="dish"><img src="./images/rajhstani-tikka.png" alt="rajhstani-tikka">
+                    <h3>Rajhstani Tikka</h3>
+                    <p>Small pieces of boneless chicken baked using skewers on a brazier called angeethi or over charcoal after marinating in spices and yogurt.</p>
+                    </div> 
+                    <div class="dish"><img src="./images/strawberry-shake.png" alt="strawberry-shake">
+                        <h3>Strawberry Shake</h3>
+                        <p>This strawberry milkshake is thick, creamy, and everything that you would hope a milkshake would be! This classic dessert is so delicious!</p>
+                        </div> 
+                    </div>
+                        <button class="contact-button"> Contact Us</button>
+                        <div class="whole-dish"><p><u>SEE THE WHOLE MENU DETAILS</u></p></div>
+</section>
+<section id="reviews">
+    <div class="reviews-title"><h2>Customer Reviews</h2></div>
+    <div class="reviews-line"></div>
+    <div class="reviewers">
+
+</div>
+</section>
+<section id="main-partners">
+    <div class="partner-title"><h3>Partners</h3></div>
+    <div class="partner-orangeline"></div>
+    <div class="partner">
+    <p>GORMET</p>
+    <p>YASIR BROAST</p>
+    <p>ALI AFZAL</p>
+    <p>NAFEES</p>
+</div>
+</section>
+</main>
+<footer id="main-footer">
```

Comment:
[OPTIONAL] Kindly add footer for your mobile view on your homepage like on the guideline.

<img width="416" alt="Screenshot 2023-02-09 at 9 53 05 PM" src="https://user-images.githubusercontent.com/25887253/217914771-f0214c4a-f987-4281-b25d-a121ca586e5a.png">